### DISCUSSION TITLE
connect to FCM for push notification

### DISCUSSION THREAD

**Discussion by windwww** [directus/directus#20090] [Feature Requests]
### Summary

it is possible to extend the notification module to FCM, so that push notifications can be done with ease

### Basic Example

it is possible to extend the notification module to FCM, so that push notifications can be done with ease

### Motivation

hybrid App can benefit most if the can happen

### Detailed Design

hybrid App can benifit most

### Must Have:
- A
- B

### Should Have:
- notification status from FCM

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

server workload maybe quite significant if notifications are too many
specification of FCM vary, update needs manpower

### Alternatives

use hook at this moment

### Adoption Strategy

it would not to be much if Directus developers do not depends on this function

**Comment by br41nslug** [comment]
For those that do not know (including me 😂) what is FCM? Sounds like an acronym but most 3 letter acronyms have multiple varying meanings.

**Reply by rijkvanzanten** [reply]
My guess is [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging) but I don't know.

**Comment by rijkvanzanten** [comment]
Closing this as the RFC isn't filled out completely.
